mycli/main.py

```diff
@@ -846,7 +846,7 @@ def format_output(self, title, cur, headers, expanded=False,
             if hasattr(cur, 'description'):
                 def sanitize(col):
                     return col if type(col) is type else text_type
-                column_types = [sanitize(converters.decoders[col[1]])
+                column_types = [sanitize(FIELD_TYPES.get(col[1]))
```

Comment:
Should it be `FIELD_TYPES.get(col[1], str)` here?

Reply:
That would be more explicit which might be better. Right now line 848 will set it to `text_type`.